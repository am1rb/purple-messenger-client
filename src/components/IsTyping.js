// @flow

import React from 'react'

import styles from './IsTyping.css'

type Props = {};


export default class IsTyping extends React.PureComponent<Props>
{
	render()
	{
		return (
			<span className={styles.label}>
				is typing...
			</span>
		)
	}
}