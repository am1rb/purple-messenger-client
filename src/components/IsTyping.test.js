// @flow

import React from 'react'
import renderer from 'react-test-renderer'

import IsTyping from './IsTyping'


describe("<IsTyping /> component tests", () => {

	it("Should not regress", () => {

		const component = renderer.create(
			<IsTyping />
		)

		expect(component.toJSON()).toMatchSnapshot()

	})

})