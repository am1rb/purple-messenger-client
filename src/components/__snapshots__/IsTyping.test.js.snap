// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IsTyping /> component tests Should not regress 1`] = `
<span
  className={undefined}
>
  is typing...
</span>
`;
